### ISSUE TITLE
Failed to download intermediary jar

### ISSUE THREAD

**Issue by gsayson** [Legacy-Fabric/legacy-looming#1]
While attempting to run my legacy Fabric mod in a development environment as a client, I ran into this stack trace:

```
Caused by: net.fabricmc.loom.util.download.DownloadException: Failed to download file from (https://maven.legacyfabric.net/net/legacyfabric/intermediary/1.8.9/intermediary-1.8.9-v2.jar) to (C:\Users\gerar\AppData\Local\Temp\legacy-intermediary-1-v22678157812200049126.jar)
	at net.fabricmc.loom.util.download.Download.error(Download.java:442)
	at net.fabricmc.loom.util.download.Download.downloadPath(Download.java:157)
	at net.fabricmc.loom.util.download.DownloadBuilder.lambda$downloadPath$1(DownloadBuilder.java:124)
	at net.fabricmc.loom.util.download.DownloadBuilder.withRetries(DownloadBuilder.java:159)
	... 191 more
Caused by: net.fabricmc.loom.util.download.DownloadException: Failed to download (https://maven.legacyfabric.net/net/legacyfabric/intermediary/1.8.9/intermediary-1.8.9-v2.jar)
	at net.fabricmc.loom.util.download.Download.error(Download.java:442)
	at net.fabricmc.loom.util.download.Download.send(Download.java:121)
	at net.fabricmc.loom.util.download.Download.doDownload(Download.java:182)
	at net.fabricmc.loom.util.download.Download.downloadPath(Download.java:154)
	... 193 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:573)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
	at net.fabricmc.loom.util.download.Download.send(Download.java:119)
	... 195 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1047)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.AsyncSSLConnection.connectAsync(AsyncSSLConnection.java:56)
	at java.net.http/jdk.internal.net.http.Http1Exchange.sendHeadersAsync(Http1Exchange.java:239)
	at java.net.http/jdk.internal.net.http.Exchange.lambda$responseAsyncImpl0$9(Exchange.java:472)
	at java.net.http/jdk.internal.net.http.Exchange.checkFor407(Exchange.java:404)
	at java.net.http/jdk.internal.net.http.Exchange.lambda$responseAsyncImpl0$10(Exchange.java:476)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl0(Exchange.java:476)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl(Exchange.java:380)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsync(Exchange.java:372)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:408)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:449)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:439)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsync0$2(MultiExchange.java:341)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(HttpClientImpl.java:157)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsync(MultiExchange.java:294)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.sendAsync(HttpClientImpl.java:654)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:552)
	... 197 more
Caused by: java.nio.channels.UnresolvedAddressException
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 213 more
```

I saw that `maven.legacyfabric.net` has been replaced by `https://repo.legacyfabric.net` in the DNS configuration, but I think it has not been reflected in the current codebase.

**Comment by BluCobalt** [comment]
`maven.legacyfabric.net` is supposed to be a redirect to `repo.legacyfabric.net/repository/legacyfabric`, both to keep backwards compatibility with old projects that expect the maven to be at `maven.` and to keep forwards compatibility in case the site the maven is hosted at ever changes. Earlier today, I finished transferring legacy fabric's infrastructure to a different vps but set up the redirect wrong. I just fixed it and it's working on my end - is it working on yours?
